src/Spice86.Core/Emulator/InterruptHandlers/Dos/DosInt2fHandler.cs

```diff
@@ -30,14 +30,13 @@ public class DosInt2fHandler : InterruptHandler {
 
     private void FillDispatchTable() {
         _dispatchTable.Add(0x15, new Callback(0x15, SendDeviceDriverRequest));
-        _dispatchTable.Add(0x43, new Callback(0x43, NoOp));
+        _dispatchTable.Add(0x43, new Callback(0x43, GetSetFileAttributes));
     }
 
     /// <summary>
-    /// This is an INT2F function that even DOSBox doesn't implement.
     /// Right now, a NOP is sufficient in order to make some games (eg. Dune 2) work.
     /// </summary>
-    public void NoOp() {
+    public void GetSetFileAttributes() {
         
```

Comment:
This could also use a `TODO` comment. 
Makes it easier to find places to work on.

Reply:
Done.